octoprint_Spoolman/modules/PrinterUtils.py

```diff
@@ -3,7 +3,11 @@
 
 import math
 
-class PrinterUtils:
+class PrinterUtils(object):
```

Comment:
Looks like I made a mistake, and what should have been just a bunch of static methods, accidentally became a class that needs an initializer. Could you please refactor `getCurrentJobFilamentUsage` so that instead of relying on internal class state, it would take necessary references as arguments?